examples/apps/presence-tracker/package.json

```diff
@@ -28,11 +28,13 @@
 		"lint": "fluid-build . --task lint",
 		"lint:fix": "fluid-build . --task eslint:fix --task format",
 		"prepack": "npm run webpack",
-		"start": "webpack serve",
-		"start:client:test": "webpack serve --config webpack.test.cjs",
+		"start": "npm run start:client",
+		"start:client": "webpack serve",
+		"start:tinylicious": "start-server-and-test tinylicious 7070 start:client",
```

Comment:
It seems strange to me to start both the tinylicious service and the client via `start:tinylicious`. I believe elsewhere (at least in our public examples), we do something like:

```json
"start": "start-server-and-test start:tinylicious 7070 start:client",
"start:client": "webpack serve",
"start:tinylicious": "tinylicious",
```

Reply:
I did it this way because the readme said to run tinylicious separately, but I just updated the readme settings instead.